modules/templates/ref-creating-templates.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * admin/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-creating-templates_{context}"]
+= Creating a template as a YAML file
+
+You can create a template by defining a `Template` object as a YAML file.
```

Comment:
There will be a separate PR that addresses adding templates where I will probably address this - this module is just to show an example of what should be defined in a template rather than how to actually add them. There are separate Jira issues for these, so I am handling them as separate issues.